src/Bicep.LangServer/Completions/BicepCompletionContext.cs

```diff
@@ -25,6 +26,22 @@ public record FunctionArgumentContext(
             FunctionCallSyntaxBase Function,
             int ArgumentIndex
         );
+        private static readonly CompositeSyntaxPattern ExpectingImportSpecification = CompositeSyntaxPattern.Create(
+            "import |",
```

Comment:
Given that Jonny has now introduced `|` as a valid token, should we pick a different char by default?

Reply:
Good catch! I copied the approach we use for the completion tests and updated the `Create` to take a cursor character as a required parameter.